tests/runner.py

```diff
@@ -520,9 +536,11 @@ def count_wasm_contents(self, wasm_binary, what):
     out = run_process([os.path.join(Building.get_binaryen_bin(), 'wasm-opt'), wasm_binary, '--metrics'], stdout=PIPE).stdout
     # output is something like
     # [?]        : 125
-    line = [line for line in out.split('\n') if '[' + what + ']' in line][0].strip()
-    ret = line.split(':')[1].strip()
-    return int(ret)
+    for line in out.splitlines():
+      if '[' + what + ']' in line:
+        ret = line.split(':')[1].strip()
+        return int(ret)
+    assert False
```

Comment:
Should probably have an error message here, as opposed to "assert failed: False"